domains/client/domain-operator/src/aux_schema.rs

```diff
@@ -35,12 +40,8 @@ const LATEST_CONSENSUS_HASH: &[u8] = b"latest_consensus_hash";
 /// - Hash(ConsensusBlock12) => Hash(DomainBlock5)
 const BEST_DOMAIN_HASH: &[u8] = b"best_domain_hash";
 
-/// Prune the execution receipts when they reach this number.
-///
-/// NOTE: `PRUNING_DEPTH` must larger than the consensus chain `ConfirmationDepthK` to avoid
-/// accidentally delete non-confirmed ER from consensus forks that can potentially become the
-/// best fork in the future
-const PRUNING_DEPTH: BlockNumber = 1000;
+/// Tracks a list of tracked domain block hash keys at a given height.
```

Comment:
From the code, it seems like we’re storing unique pairs of best domain block hashes and latest consensus block hashes, for a given height. It might be helpful to expand the comment with a few words to say why they’re being tracked.

We could also say “stores”?

```suggestion
/// Holds a list of tracked best domain block hashes and latest consensus block hashes, at a given height.
```